lib/flags.ml

```diff
@@ -82,3 +82,11 @@ let get_inline_level () = !inline_level
 
 let profile_ltac = ref false
 let profile_ltac_cutoff = ref 2.0
+
+let native_compiler = ref None
+let get_native_compiler () = match !native_compiler with
+| None -> assert false
```

Comment:
This kind of stuff seems pretty horrible I think; just set it to a reasonable default instead of breaking all people linking to Coq in a minor release.

Reply:
PR welcome. AFAIK we don't promise OCaml runtime backwards compatibility between minor releases, we just do a best effort not to break the API.